File: src/contests/cg/puzzle/swift/google-interview---the-two-egg-problem.swift
// 🎮 CodinGame Puzzle - google-interview---the-two-egg-problem
// https://www.codingame.com/training/hard/google-interview---the-two-egg-problem

let n = Int(readLine()!)!
var k = 0
while k * (k + 1) / 2 < n {
    k += 1
}
print(k)

File: src/contests/cg/puzzle/java/google-interview---the-two-egg-problem.java
// 🎮 CodinGame Puzzle - google-interview---the-two-egg-problem
// https://www.codingame.com/training/hard/google-interview---the-two-egg-problem

import java.io.*;

class Solution {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        long n = Long.parseLong(br.readLine().trim());
        long k = 0;
        while (k * (k + 1) / 2 < n) k++;
        System.out.println(k);
    }
}
